### ISSUE TITLE
Fulfillment/Bounty Stats Analytics Endpoint

### ISSUE THREAD

**Issue by villanuevawill** [Bounties-Network/BountiesAPI#50]
We want to show the cumulative number of unique bounty issuers and cumulative bounty fulfillers over time.  You can view the current [beta analytics dashboard](https://analytics.bounties.network). This task only requires updating the API. A separate task will be created around the frontend.

# Requirements
- The endpoint should return cumulative numbers on unique issuer addresses and fulfiller addresses
- Cumulatives should be added to both daily and weekly records

# Review
- Should correspond regularly and give an initial plan of action
- Key questions posted here
- Conversational and longer discussions should be directed to the bounties network slack in this [channel](https://join.slack.com/t/bountiesnetwork/shared_invite/enQtMzA2Mjk3MzAzODQwLTZjN2UxMmU5MWYxZTVmMmM4OGNjZDRiMDgwYTVhOTIwYmQ4MjVlMjNkZjYzOTE4MWI4OTFhOWE4ZTUzN2MyNWY)

# Qualifications
- This is an intermediate task and the patterns are already in place. A good github history and prior experience with python, django, docker are required/encouraged

# Deadline
- Please only take this issue if you have the capacity to work on this ACTIVELY over the next week. We'd like this finished quickly and do not want someone to take it who will passively work on it.

Before starting work, please communicate your plan and availability along with your intention to take the task.  After initial communication, I'll approve the best candidate to take the task.
